great_expectations/data_context/store/gx_cloud_store_backend.py

```diff
@@ -664,3 +624,107 @@ def _validate_key(self, key) -> None:
             raise TypeError(
                 f"The provided resource_type {resource_type} is not a valid GXCloudRESTResource"
             )
+
+    @classmethod
+    def construct_versioned_url(
+        cls,
+        base_url: str,
+        organization_id: str,
+        resource_name: str,
+        id: Optional[str] = None,
+    ) -> str:
+        """Construct the correct url for a given resource."""
+        version = cls._ENDPOINT_VERSION_LOOKUP.get(resource_name, EndpointVersion.V0)
+
+        if version == EndpointVersion.V1:
+            url = urljoin(
+                base_url,
+                f"api/v1/organizations/{organization_id}/{hyphen(resource_name)}",
+            )
+            if id:
+                url = f"{url}/{id}"
+        else:  # default to EndpointVersion.V0
+            url = urljoin(
+                base_url,
+                f"organizations/{organization_id}/{hyphen(resource_name)}",
+            )
+            if id:
+                url = f"{url}/{id}"
+        return url
+
+    @classmethod
+    def construct_versioned_payload(  # noqa: PLR0913
+        cls,
+        resource_type: str,
+        organization_id: str,
+        attributes_key: str,
+        attributes_value: Union[dict, Any],
+        resource_id: str | None = None,
+        **kwargs: dict,
+    ) -> dict:
+        """Construct the correct payload for the cloud backend.
+
+        Arguments `resource_type`, `resource_id`, and `attributes_value` of type Any
+        are deprecated in GX V1, and are only required for resources still using V0 endpoints.
+        """
+        version = cls._ENDPOINT_VERSION_LOOKUP.get(resource_type, EndpointVersion.V0)
+        if version == EndpointVersion.V1:
+            if isinstance(attributes_value, dict):
+                payload = {**attributes_value, **kwargs}
+            elif attributes_value is None:
+                payload = kwargs
+            else:
+                raise TypeError(
```

Comment:
this coverage annotation is incorrect; see `test_construct_json_payload_raises_error_with_V1_resource_and_wrong_attributes_value_type`

Reply:
Seems like this coverage report is creating a lot of unnecessary noise 🤔

Reply:
That should be fixable.